Sample_v3/Samples/SimpleChat/SimpleChatViewController.swift

```diff
@@ -132,6 +132,11 @@ final class SimpleChatViewController: UITableViewController, ChatChannelControll
         
         cell.backgroundColor = message.localState == nil ? .white : .lightGray
         
+        if #available(iOS 13, *) {} else {
+            let longTapGestureRecognizer = UILongPressGestureRecognizer(target: self, action: #selector(didTapMessage(_:)))
+            cell.addGestureRecognizer(longTapGestureRecognizer)
+        }
```

Comment:
Can you add a comment explaining what's happening here and why it's <iOS13 only?

Reply:
iOS13> : We can use context menus
<iOS13 : Context menus not available, we have to use long tap + action sheet

Reply:
Yeah, but can you put this comment to the code? :D